kbfsblock/bserver_errors.go

```diff
@@ -131,10 +132,13 @@ func (e BServerErrorOverQuota) ToStatus() (s keybase1.Status) {
 
 // Error implements the Error interface for BServerErrorOverQuota.
 func (e BServerErrorOverQuota) Error() string {
-	if e.Msg == "" {
-		return "BServer: user has exceeded quota"
+	msg := e.Msg
+	if msg == "" {
+		msg = "BServer: user has exceeded quota"
 	}
-	return "BServerErrorOverQuota{" + e.Msg + "}"
+	return fmt.Sprintf(
+		"BServerErrorOverQuota{Msg: %s, Usage: %d, Limit: %d, Throttled: %t}",
```

Comment:
Hmm that would break with the usual pattern. What about letting `e.Msg` be empty and just put it in the message?